docs/multi-cluster-mesh/active-passive/multi-cluster-active-passive.md

```diff
@@ -0,0 +1,106 @@
+# Single mesh multi cluster multiple networks with Istio operator
+
+## Objective:
+Setup three clusters (one active – two passive) in a single mesh multi network (two in the same network, one in a different network) setup with Istio operator.
+
+## Setup: 
+
+### Steps:
+
+#### Active setup:
+1. Install cluster registry controller:
+```
+helm install --namespace=cluster-registry --create-namespace cluster-registry-controller deploy/charts/cluster-registry  --set image.tag=v0.1.9 --set localCluster.name=demo-active --set network.name=network1
+```
+2. Install istio operator:
```

Comment:
I'd suggest to do the cluster registry setup first on both clusters and the cluster cr and secret swap and install istio operator only after.